devops/README.md

```diff
@@ -0,0 +1,71 @@
+# OSConfig DevOps
+This folder contains various folders in use by OSConfig's Engineering Systems (Build/Packaging/Test Infrastructure). This readme serves to describe the various components as they have been created with reusability in mind.
```

Comment:
>Build/Packaging/Test Infrastructure [](http://example.com/codeflow?start=79&length=35)

Can we have links for these? Also, separated by comma instead of slash

Reply:
no links as the assets e2e assets are shared. ill add more description to the folders contents as well.